sda-orch/values.yaml

```diff
@@ -67,6 +67,25 @@ broker:
     ingest: "ingest"
     mappings: "mappings"
 
+# Datacite and REMS depend on one another
+# if we set one the other one needs to be configured
+# By default the dataset IDs are not exposed and are assigned
+# internally with no visibility to the user.
+datasetID:
+  external: false
+  # If `useCustomConfig` is enabled it expects a file
+  # with the name `config.json` so that it can load it as configuration from it.
+  useCustomConfig: false
```

Comment:
Another way to do this is to template the custom config here
``` 
customConfig:
  key1: value1
  key2: value2
```
And then use `toJson` when rendering the config map

Reply:
I would leave that for next version, the template is a tad complex especially when it comes with translations.
I can probably work out a better structure but it could come with a next version of sda-orch.